### DISCUSSION TITLE
Retrieve domain associated with deleted team

### DISCUSSION THREAD

**Discussion by JustinASmith** [vercel/community#5896] [Help]
### Summary

Hi, I previously had the domain "roundfinder.com" associated with a team "bitsof-xyz"; however, I deleted the team and no longer have access to my "roundfinder.com" domain. Would someone be able to help me gain access to this domain under my personal account?

Thank you!

### Steps to Reproduce

1. Create team
2. Create and deploy app associated with team
3. Transfer in the domain from outside registrar
4. Assign domain to deployed app in step 2
5. Transfer app to personal account
6. Delete team

**Comment by JustinASmith** [comment]
This is not the most exciting thing to some people, but I need this resolved. If anyone can expedite this, I would be very grateful!

**Reply by JustinASmith** [reply]
@amyegan

**Comment by amyegan** [comment]
Hey @JustinASmith. You should be able to re-add this to your Hobby account via the "Add" button from the Domains page at https://vercel.com/dashboard/domains

If you run into trouble adding it to your Hobby team, you can also try adding directly to your personal account at https://vercel.com/account/domains

Please give that a try and let me know whether or not it works for you
